pkg/cmd/server/kubernetes/network/sdn_linux.go

```diff
@@ -29,6 +33,11 @@ func NewSDNInterfaces(options configapi.NodeConfig, networkClient networkclient.
 	// SDN's hostport handling when run under CRI-O.
 	enableHostports := !strings.Contains(runtimeEndpoint, "crio")
 
+	sdnInformers := sdncommon.SDNInformers{
```

Comment:
The SDNInformers type is useful to the SDN code itself for passing to RegisterSharedInformer, but I don't think you should use it from outside code like this; just pass the two informers separately.